packages/DataHeader/stories/DataHeader.stories.mdx

```diff
@@ -0,0 +1,154 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+import { getStoryName } from "storybook/storyTree";
+import Readme from "../README.md";
```

Comment:
You import the README, but then don't render it.  Ideally we can save ourselves some work in the Docs page by reusing that content, which should demonstrate the basic use cases. Then the Docs page can illustrate more interesting/advanced use cases.